C-Plus-Plus/cp/Space_Array.cpp

```diff
@@ -0,0 +1,72 @@
+/* PROBLEM LINK: https://www.codechef.com/MARCH21C/problems/SPACEARR
+*/
+
+/*
+The rules of the game are as follows:
+1-Initially, a sequence a1,a2,…,aN is given.
+2-The players alternate turns.
+3-In each turn, the current player must choose
+an index i and increment ai, i.e. change ai to ai+1.
+4-Afterwards, if there is no permutation p1,p2,…,pN of the
+integers 1 through N such that ai=pi holds for each valid
+ i, the current player loses.
+5-Otherwise, the game continues with the next turn.
+*/
+
+// Code
+#include<bits/stdc++.h>
+using namespace std;
+int main() {
```

Comment:
Use a different function for the main logic of the code. Use main() function for the driver code